src/audio/module_adapter/library/native_system_agent.c

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
```

Comment:
Please use proper commit message prefix. E.g 

``module_adapter: native_system_agent: Introduce call and interface``

Reply:
ack. @pjdobrowolski can you do this in next uprev. Thanks !

Reply:
done